script/Ciel/Day 8/08_01_CIEL08_1D.txt

```diff
@@ -269,158 +290,190 @@ I want to know more about her.
 [sha:bee58b0ec213cec471fecc2fab3266e8a8bce963]{
 -- Page 120, Offset 18560.
 -- その結果が、
-As a result―――
+And because of that―――
 }
 [sha:2809088b42670d875ee620adceb3890a89649d27]{
 -- Page 120, Offset 18561.
 -- 何の言い逃れもできない、このような結末だった。
-Without excuse or escape, this is how it all ends.
+// Accompanied by 17 pieces CG change as target of 'this'
+This is the result.
+ An outcome that I cannot make excuses for.
```

Comment:
disagree. ross's version is fine

Reply:
'An outcome that I cannot make excuses for.' just doesn't vibe with me at all. It lacks the dramatic quality it deserves.

> This is the result.
> I cannot make any excuses for this outcome.

Would at least make it feel a little better in that sense, imo.

Reply:
Seems like there's 2 things at play here - the first is the difference between 言い逃れ[る]'s interpretation as 'unavoidable' as per Petrikow's initial comment vs 'I have nothing to say for my actions', which is what I'd read it as. 
Second is whether or not it's got the Vibes, which is obv more subjective but my intention with this mildly more formal/stuffy phrasing is with the mental image of him being before a judge.

Reply:
I think the 言い逃れ is about how he can't avoid responsibility for his actions, so in that sense, 'I cannot make excuses for' hits in the same ballpark, but it just doesn't sound right. It's not grave enough for someone who has committed murder. That's the Vibes, I suppose.

> This is the result. Unmistakable and undeniable.

Maybe?

Reply:
That reads more to me like he's saying "you can't deny that this happened" rather than "I cannot deny that I did this". If we want to sound serious but get the idea across I feel like we need to depart a bit from literal 'inexcusable', e.g. something like
> This is the result. An atrocity[action if we don't want to be so bold] I cannot atone for.